src/main/assets/scss/main.scss

```diff
@@ -16,6 +16,53 @@
   font-size: 1.1495rem;
 }
 
+.courts-table-header-inactive {
+
+  color: govuk-colour('blue', $legacy: 'blue-3');
+
+  &::after {
```

Comment:
This is the only styling that's different between the 3 new classes. Could we remove the repeated parts into a separate class?

Reply:
I have tried splitting it out previously, but it seems to create styling issues and such, can look into it further if you think its a priority